examples/nbproject/project.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://www.netbeans.org/ns/project/1">
+    <type>org.netbeans.modules.cnd.makeproject</type>
+    <configuration>
+        <data xmlns="http://www.netbeans.org/ns/make-project/1">
+            <name>CppUTest</name>
+            <c-extensions>c</c-extensions>
+            <cpp-extensions>cpp</cpp-extensions>
+            <header-extensions>h</header-extensions>
+            <sourceEncoding>UTF-8</sourceEncoding>
+            <make-dep-projects/>
+            <sourceRootList>
+                <sourceRootElem>src/CppUTest</sourceRootElem>
+                <sourceRootElem>src/CppUTestExt</sourceRootElem>
+                <sourceRootElem>src/Platforms</sourceRootElem>
+                <sourceRootElem>include</sourceRootElem>
+                <sourceRootElem>examples/AllTests</sourceRootElem>
+                <sourceRootElem>../include/CppUTest</sourceRootElem>
+                <sourceRootElem>../include/CppUTestExt</sourceRootElem>
+                <sourceRootElem>../include/Platforms</sourceRootElem>
+                <sourceRootElem>AllTests</sourceRootElem>
+                <sourceRootElem>ApplicationLib</sourceRootElem>
+                <sourceRootElem>../src/CppUTest</sourceRootElem>
+                <sourceRootElem>../src/CppUTestExt</sourceRootElem>
+                <sourceRootElem>../src/Platforms</sourceRootElem>
+                <sourceRootElem>../tests</sourceRootElem>
+            </sourceRootList>
+            <confList>
+                <confElem>
+                    <name>Debug</name>
+                    <type>1</type>
+                </confElem>
+            </confList>
+            <formatting>
+                <project-formatting-style>false</project-formatting-style>
+            </formatting>
+        </data>
+    </configuration>
+</project>
```

Comment:
Usually it would be better to keep them separate. Larger pull requests take me longer to check and thus tend to get delayed. Smaller pull requests are easier and will be integrated faster.